lesson6.js

```diff
@@ -1,7 +1,13 @@
 const ul = document.getElementById('js-list');
+const loadingTime = document.createElement('img');
+loadingTime.src = "loading-circle.gif";
+loadingTime.id = 'loading-img'
+ul.appendChild(loadingTime);
+const loading = document.getElementById('loading-img');
```

Comment:
IMO
Instead of declaring them globally, it is better to declare them where they will be used, which will reduce bugs.